pkg/storaged/dialog.jsx

```diff
@@ -1068,6 +1070,7 @@ export const BlockingMessage = (usage) => {
 };
 
 const UsersPopover = ({ users }) => {
+    const mock = useMock()?.busy;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17654-20230322-115306-eab6c63e-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)